streamwebs_frontend/streamwebs/views.py

```diff
@@ -24,11 +24,203 @@
     TransectZone, Canopy_Cover, CameraPoint, PhotoPoint,
     PhotoPointImage, Soil_Survey, Resource, School)
 
+from djqscsv import render_to_csv_response
+import csv
 import json
 import copy
 import datetime
 
 
+def export_wq(request, site_slug):
```

Comment:
Also you can drop the `request` parameter, you don't use it in any of the export views.